tests/System/SystemTestCase.php

```diff
@@ -49,6 +49,7 @@ public function phpbench($command, $workingDir = '.')
     {
         chdir($this->getWorkingDir($workingDir));
         $bin = __DIR__ . '/../../bin/phpbench --verbose';
+        $bin = str_replace('/', DIRECTORY_SEPARATOR, $bin);
```

Comment:
Ah ok, yes your right. I would probably go for `implode(DIRECTORY_SEPARATOR, array('path', 'to', 'something'))` in that case.